util-taglib/src/com/liferay/taglib/portletext/RuntimeTag.java

```diff
@@ -261,8 +261,9 @@ public static void doTag(
 					"the-application-cannot-include-itself-or-a-portlet-of-" +
 						"same-type");
 
-				httpServletRequest.setAttribute(
-					"liferay-portlet:runtime:errorMessage", errorMessage);
+				pageContext.setAttribute(
```

Comment:
@yuhai The commit message is confusing. Please try to provide a short and clear description with reference.